lib/pry-theme/commands.rb

```diff
@@ -233,7 +233,7 @@ def json_body(address)
     end
 
     def windows?
-      (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
+      Pry::Helpers::Platform.windows?
```

Comment:
Do we need to keep the whole method? Why not just to inline the call?

Reply:
Right. I will fix this.